gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt

```diff
@@ -83,24 +102,37 @@ class ReaperClassLoadClassVisitor(
     return mv?.let {
       val sig = "$className.$name$descriptor"
       logger.info("Processing method: $sig")
-      ReaperClassLoadMethodVisitor(api, mv, className, name)
+      ReaperClassLoadMethodVisitor(api, mv, className, name, logger, writer)
     }
   }
 }
 
+fun longToBase64(hash: Long): String {
+  val buf = ByteArray(8)
+  for (i in 0..7) {
+    buf[i] = ((hash shr i * 8) and 0xFFL).toByte()
+  }
+  val hashAsBase64 = Base64.getEncoder().encode(buf).toString(Charsets.UTF_8)
+  return hashAsBase64
+}
+
 class ReaperClassLoadMethodVisitor(
   api: Int,
   methodVisitor: MethodVisitor,
   private val className: String,
-  // private val writer: PrintWriter,
   private val name: String?,
+  private val logger: Logger,
+  private val writer: PrintWriter,
 ) : MethodVisitor(api, methodVisitor) {
   override fun visitCode() {
     super.visitCode()
     if (name == "<clinit>" || name == "<init>") {
       val signature = "L" + className.replace(".", "/") + ";"
       val hashedSignature = topLong(toSha256(signature))
 
+      writer.write("${className}\t${hashedSignature}\t${longToBase64(hashedSignature)}\t${signature}\t${name}\n")
```

Comment:
I think you'll need to add to a mutable struct and then save as part of the task like: https://github.com/EmergeTools/emerge-android/blob/b567f2c1bcf1b06d8e345980dd41c129a95704fa/gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/transform/FindPreviewsAcrossProjects.kt#L41